Python/Multiplicative Inverse.py

```diff
@@ -0,0 +1,44 @@
+def extended_euclidean(p, q):
+    """(int, int) -> list
+
+    Return the list : [greatest_common_divisor, multiplicative_inverse(1), multiplicative_inverse (if multiplicative_inverse(1) is negative)]
+
+    >>> extended_euclidean(10, 23)
+    [1, [7]]
+
+    >>> extended_euclidean(11, 2)
+    [1, [-5, -3]]
+    """
+    #Initalizing table of Extended Euclidean algorithm
+    table = {'quotent':0, 'remainder_temp1':max(p, q), 'remainder_temp2':min(p, q), 'remainder':0, 'multiplicative_inverse1':0, 'multiplicative_inverse2':1, 'multiplicative_inverse':0}
+
+    while(True):
+        try:                                                                      #Break condition for loop

```

Comment:
The comments are not properly formatted. Comments are given two spaces between code and # and one space between # and actual text. For instance,
```python
print('Something')  # This is a comment
```

Correct this for all instances in the code